Gopkg.toml

```diff
@@ -26,6 +26,11 @@ required = [
   # HEAD as of 2919-06-26
   revision = "677d3d17e84b7337f006220d6bb465af4f416f81"
 
+[[override]]
+  name = "github.com/knative/serving"
+  # HEAD as of 2919-06-26
+  revision = "50cdf8762deae58ed89763adef26a88e7efecac7"
```

Comment:
no I am using `knative.dev/pkg`

Reply:
IIUC, you can't use version 0.7.0 because you have an existing dependency on `knative.dev/pkg` and serving 0.7 uses `github.com/knative/pkg`. I suggest using `github.com/knative/pkg` to avoid this and depending on serving 0.7, but if you don't want to do that, add a comment explaining why this is a revision constraint so future maintainers know when it can be updated to a version constraint.

Reply:
I plan to not integrate 0.7 with this code. It will dep on master, aka 0.8 and above.

Reply:
Is your plan for it to ALWAYS depend on master, or will it depend on 0.8 when 0.8 is released?

Reply:
Plan is to always depend on serving and eventing master.